src/design-tokens/mixins.css

```diff
@@ -7,15 +7,14 @@
 }
 
 @define-mixin textLink {
-  /* TODO: inherit size */
```

Comment:
This was added in the `Button` CSS.

Reply:
should we also add it here? for whatever other places might use this mixin?

Reply:
Oo, good call 👍 I moved it to the mixin, thanks!